src/App.js

```diff
@@ -0,0 +1,30 @@
+import logo from './logo.svg';
+import './App.css';
+
+function App() {
+  return (
+    <div className="App">
+      <header className="App-header">
+        <img src={logo} className="App-logo" alt="logo" />
+        <p>
+          Edit
+          {' '}
+          <code>
+            src/App.js
+          </code>
+          and save to reload.
+        </p>
+        <a
+          className="App-link"
+          href="https://reactjs.org"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          Learn React
+        </a>
+      </header>
+    </div>
+  );
+}
```

Comment:
- [ ] Since all the information here was bootstrapped by `create-react-app` we can remove it and add a message instead.
```JS
function App() {
  return (
    <h1>Hello World!</h1>
  )
}
```